index.html

```diff
@@ -332,7 +332,7 @@ <h2>The <dfn>ThingDescription</dfn> type</h2>
         <a>TD</a>.
       </p>
       <div>
-        To <dfn>expand a TD</dfn> given |td:ThingDescription|, run the following steps:
```

Comment:
I think this <dfn> element should be kept, is it @zolkis?

Reply:
Yes.